### DISCUSSION TITLE
Set Label of DatePicker

### DISCUSSION THREAD

**Discussion by seikosantana** [AvaloniaCommunity/Material.Avalonia#296] [Q&A] [answered]
Is there a way to set the label of DatePicker so we can display something else instead of `Date`?
![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/35630407/3f225bfd-5310-454b-b33f-62016186b927)

I'm relatively new to Avalonia and XAML.
I tried overriding the style for that using 
```xml
<UserControl.Styles>
        <Style Selector="DatePicker /template/ TextBlock">
            <Setter Property="Background" Value="Red"></Setter>
        </Style>
</UserControl.Styles>
```
on my user control containing the DatePicker and I dont see it changing background.

Thanks the for awesome repo

**Comment by SKProCH** [comment, marked as answer]
Hello, @seikosantana 

You can actually set `assists:TextFieldAssist.Label` for DatePicker itself. 
So:
```xaml
<DatePicker assists:TextFieldAssist.Label="Your label here" ... />
```

**Reply by seikosantana** [reply]
Thanks! I didn't notice we have a demo for it. It has all examples i needed.
